lib/cmd/common/admin-storeitems.js

```diff
@@ -185,11 +185,11 @@ function update(guid, params, cb) {
 }
 
 function addIcon(guid, iconFile, cb) {
-  fhreq.uploadFile("/box/srv/1.1/admin/storeitem/uploadbinary", iconFile, {"type":"icon", "guid":guid}, "application/octet-stream" , cb);
+  fhreq.uploadFile("/box/srv/1.1/admin/storeitem/uploadbinary", iconFile, {"type":"icon", "guid":guid}, cb);
```

Comment:
I think that we don't have validation of the type of file which will be upload and we can't ensure it in this way, so in my point of view, we should not define it into the code and we can make this method more simple and reduce the number of parameters. Is it make sense? What do you think could we move forward in this way?